cmd/kube-controller-manager/app/core_test.go

```diff
@@ -141,3 +144,204 @@ func TestController_DiscoveryError(t *testing.T) {
 		}
 	}
 }
+
+func TestGetNodeCIDRMaskSizes(t *testing.T) {
```

Comment:
single-stack IPv4 no flags with`node-cidr-mask-size`-> default mask 24
single-stack IPv6 no flags with`node-cidr-mask-size` -> default mask 64
dual-stack only node-cidr-mask-size-ipv4 -> default ipv6 mask 64
dual-stack only node-cidr-mask-size-ipv6 -> default ipv4 mask 24
dual-stack no flags with`node-cidr-mask-size`  -> default ipv4 mask 24 and ipv6 mask 64

Reply:
> /hold
> @SataQiu I think we are missing test cases for the defaults mask, aren't we?
> `ipv4Mask, ipv6Mask := defaultNodeMaskCIDRIPv4, defaultNodeMaskCIDRIPv6`

Good catch! 👍